dataframe-geo/src/main/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/IntegrationGeo.kt

```diff
@@ -0,0 +1,84 @@
+@file:Suppress("INVISIBLE_MEMBER", "INVISIBLE_REFERENCE")
+
+package org.jetbrains.kotlinx.dataframe.jupyter
+
+
+import org.jetbrains.kotlinx.dataframe.codeGen.CodeWithConverter
+import org.jetbrains.kotlinx.dataframe.geo.GeoDataFrame
+import org.jetbrains.kotlinx.dataframe.geo.GeoFrame
+import org.jetbrains.kotlinx.dataframe.geo.MultiPolygonGeoFrame
+import org.jetbrains.kotlinx.dataframe.geo.PolygonGeoFrame
+import org.jetbrains.kotlinx.dataframe.impl.codeGen.ReplCodeGeneratorImpl
+import org.jetbrains.kotlinx.jupyter.api.FieldHandler
+import org.jetbrains.kotlinx.jupyter.api.FieldHandlerExecution
+import org.jetbrains.kotlinx.jupyter.api.KotlinKernelHost
+import org.jetbrains.kotlinx.jupyter.api.VariableName
+import org.jetbrains.kotlinx.jupyter.api.libraries.FieldHandlerFactory
+import org.jetbrains.kotlinx.jupyter.api.libraries.JupyterIntegration
+import kotlin.reflect.KProperty
+import kotlin.reflect.KType
+import kotlin.reflect.full.isSubtypeOf
+import kotlin.reflect.typeOf
+
+internal class IntegrationGeo : JupyterIntegration() {
+
+    // TODO make internal in core and use here
+    private fun KotlinKernelHost.execute(codeWithConverter: CodeWithConverter, argument: String): VariableName? {
+        val code = codeWithConverter.with(argument)
+        return if (code.isNotBlank()) {
+            val result = execute(code)
+            if (codeWithConverter.hasConverter) {
+                result.name
+            } else {
+                null
+            }
+        } else {
+            null
+        }
+    }
+
+    // TODO make internal in core and use here
+    private fun KotlinKernelHost.execute(
+        codeWithConverter: CodeWithConverter,
+        property: KProperty<*>,
+        type: KType,
+    ): VariableName? {
+        val variableName = "(${property.name}${if (property.returnType.isMarkedNullable) "!!" else ""} as $type)"
+        return execute(codeWithConverter, variableName)
+    }
+
+    override fun Builder.onLoaded() {
```

Comment:
what does this jupyter integration do exactly? could you add some docs?

Reply:
Hmm, ok, but there's no docs for core integration

Reply:
Maybe a test like below will make things clearer
for reference:
org.jetbrains.kotlinx.dataframe.jupyter.DataFrameJupyterTest and org.jetbrains.kotlinx.dataframe.jupyter.CodeGenerationTests

test:
```
val geo = **create geo**
val updatedGeo = geo.update { add("col") { 1 }  }
updatedGeo.df.col
```

Reply:
> Hmm, ok, but there's no docs for core integration

That's not a good argument haha. We also need kdocs there :)

Reply:
Done

Reply:
thanks!